comp120-evaluation.tex

```diff
@@ -0,0 +1,58 @@
+
+\section{Reading and Understanding Peers' Code}
+Due to a lack of experience in reading and writing code, I am experiencing difficulty comprehending peers' code. Despite being able to understand small sections, I struggle to see how it all fits together. I immediately noticed that this makes constructing useful comments in peer review sessions difficult.
+
+If this skill remains undeveloped for the forthcoming collaborative projects, I will struggle to provide feedback and understand the code I am contributing to. This is equally important if I am going to collaborate with others professionally as a games developer.
+
+To develop this skill, I will ensure that I read peers' code at least once each week and predict its outcome before running the program. After one month, I will review how frequently these predictions were accurate.
+
+\section{Time Management}
+I have struggled with time management due to my perfectionistic nature combined with an extended period out of education. The most significant issue for me is spending more time than necessary perfecting and adding unnecessary features to a single project whilst neglecting others.
+
+It is crucial that I develop this skill in preparation for the increased workload as the course progresses. Similarly, it is important that I recognise when something is complete, in order to focus on other tasks. If I become an independent games developer, this will ensure that development time remains sensible. After using Trello with my Kivy project, I have noticed that it helps with this aspect. I think that this is because it breaks the project down and allows me to see when I have completed each task. It also ensures that I focus only on the tasks in the sprint plan.
+
+I will address this issue by creating a flexible study timetable at the beginning of each week. I will also set up a Trello board for each project and utilise sprint planning. In two months, I will assess whether this allowed me to comfortably meet assignment deadlines. I will also measure how often I deviated from the sprint plan by comparing my commit logs with the Trello board.
+
+\section{Communication}
+I experience significant difficulties with communication, primarily due to anxiety. My reluctance to ask for feedback, help and clarification has negatively impacted my studies. Attempting to resolve every issue myself has resulted in unnecessary work, high stress levels and reduced productivity close to assignment deadlines.
+
+I have realised that I waste considerable amounts of time pondering uncertainties or being stuck. Consequently, my struggles in this area adversely affect the effectiveness of my time management. Therefore, addressing this issue is crucial if I am to cope with an increased workload. Additionally, if I become an independent games developer, I must learn to communicate effectively and be willing to ask for help in order to learn new tools and participate in the open source community.
```

Comment:
Consider revising, or perhaps even removing, the clause: "my struggles in this area adversely affect the effectiveness of my time management".